core/server/middleware/error-handler.js

```diff
@@ -88,6 +89,12 @@ module.exports = function errorHandler(err, req, res, next) {
         err = err[0];
     }
 
+    if (!(err instanceof errors.GhostError)) {
+        err = new errors.GhostError({
+            err: err
+        });
```

Comment:
Did a quick test (deleted core/server/views/default.hbs) and answered my own question. This works great 👍 🎉